### ISSUE TITLE
Revamp improvements to search page

### ISSUE THREAD

**Issue by jordan1776** [liftoff-app/liftoff#105]
### Is your feature request related to a problem? Please describe.

I feel like the search page needs some attention to match the aesthetics of the rest of the app and functionally. At the moment it works pretty good, but the UI is lacking.

### Describe the solution you'd like

First of all the "search page" and "search results" concept had to go. Should be 1 page with the search populating the space below. 

 Suggestions:
- It should be visually similar to the communities page (use the same search box with x button).
- search button should be always interactable (no solid text)
- The sort button should an icon inline with search bar
- instance drop-down should be an icon (with selected instance image shown)
- Fill the space below the unsearched field with search history? 

### Describe alternatives you've considered

Order and positions of items could be changed.

### Additional context

Quick and dirty mookup of what I mean
![Screenshot_20230626-201058~3](https://github.com/liftoff-app/liftoff/assets/6100101/9525d3f0-bcd0-4cf0-9d48-5a2e820dbaf7)

**Comment by jordan1776** [comment]
Minor note: Instance, search bar, sort button would probably look more visually balanced (and have the back arrow removed)

**Comment by mario-g98** [comment]
Maybe shorten the communities descriptions or remove them entirely and instead show number of subscribers and the instance they are in?

And maybe add a generic picture , so that it doesn't look empty when one is missing.

![Screenshot_2023-07-03-10-30-34-05_74ab5dcdef147bb53a2c1aad56578295](https://github.com/liftoff-app/liftoff/assets/49582442/e788f68e-08c2-4ec9-bd94-8a80b8ced771)

**Comment by jordan1776** [comment]
yes either show only the first few lines of the description or the first paragraph would work.
